src/providers/config-provider.tsx

```diff
@@ -18,11 +19,90 @@ export const ConfigProvider = ({ children }: IConfigProviderProps) => {
     const value = configFile.customizations[id];
     return value;
   };
+
+  const parseConfigKeys = () => {
+    const keys = Object.keys(configFile.customizations);
+    const result = keys.map((key) => {
+      const [page, component, element] = key.split('/');
+      return { page, component, element };
+    });
+    return result;
+  };
+
+  const customSort = (
+    a: {
+      page: string;
+      component: string;
+      element: string;
+    },
+    b: {
+      page: string;
+      component: string;
+      element: string;
+    }
+  ) => {
+    if (a.page !== '*' && b.page === '*') return -1;
+    if (a.page === '*' && b.page !== '*') return 1;
+    if (a.component !== '*' && b.component === '*') return -1;
+    if (a.component === '*' && b.component !== '*') return 1;
+    if (a.element !== '*' && b.element === '*') return -1;
+    if (a.element === '*' && b.element !== '*') return 1;
+    return 0;
+  };
+
+  const getUiKitConfig = ({
```

Comment:
imo, this function might be too complex you may talk with @chaiwattsw about this

Reply:
Yes. it is too complex to dynamically get  correct config with high priority ( not start * ) and low priority ( star * ) . !_!